content/documentation/pages/1-installation/2-cloudfoundry/1-cf-cli.md

```diff
@@ -1,14 +1,14 @@
 ---
-path: 'installation/cloudfoundry/'
-title: 'Cloud Foundry'
-description: 'Install Data Flow on your Cloud Foundry'
+path: 'installation/cloudfoundry/cf-cli'
```

Comment:
For discussion.  Can we make this more concise, like Gunnar's example or is that the wrong direction.

Reply:
What is gunnar's example?

Reply:
Located here: 
http://localhost:8000/documentation/master/batch-developer-guides/batch/data-flow-simple-task-cloudfoundry/